desktop/src/index.jsx

```diff
@@ -20,10 +21,12 @@ export const theme = createMuiTheme({
 })
 
 ReactDOM.render(
-  <Provider store={store}>
-    <MuiThemeProvider theme={theme}>
-      <App />
-    </MuiThemeProvider>
-  </Provider>,
+  <HashRouter>
```

Comment:
The reason that we have to do this is because we aren't using clientside rendering.

https://reactrouter.com/en/main/start/tutorial#client-side-routing

I am setting the history

<img width="858" alt="CleanShot 2023-08-16 at 21 14 42@2x" src="https://github.com/joshuawootonn/time-track/assets/20950876/514db80f-8abd-44a8-87f4-645b535c8f19">


And then calling these functions from `<button />`s. These should be `<a />` tags.

For now this is ok. But I'll create a ticket for you to go through and update all these.

Reply:
https://trello.com/c/v8QSCQ5p